<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWindow</class>
 <widget class="QWidget" name="TestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbImg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string>lbImg</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbText1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>341</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>lbText1</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>341</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbText2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>lbText2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
